VultisigApp/VultisigApp/View Models/SendCryptoVerifyViewModel.swift

```diff
@@ -36,6 +36,35 @@ class SendCryptoVerifyViewModel: ObservableObject {
     func amount(for coin: Coin, tx: SendTransaction) -> BigInt {
         return tx.amountInRaw
     }
+    
+    func blowfishEVMTransactionScan(tx: SendTransaction) async throws -> BlowfishResponse? {
```

Comment:
We also need this information on `KeysignMessageConfirmView` , on pair device , it is important to scan the tx

Reply:
Working on it

Reply:
![image](https://github.com/user-attachments/assets/c08414b6-31cf-42c5-8dbb-1700cfefabab)